src/operator/softmax_activation-inl.h

```diff
@@ -29,9 +29,9 @@ enum SoftmaxActivationOpType {kInstance, kChannel};
 
 struct SoftmaxActivationParam : public dmlc::Parameter<SoftmaxActivationParam> {
   // use int for enumeration
-  int type;
+  int act_type;
   DMLC_DECLARE_PARAMETER(SoftmaxActivationParam) {
-    DMLC_DECLARE_FIELD(type)
+    DMLC_DECLARE_FIELD(act_type)
```

Comment:
For activation, we use ```act_type```, other word may be like ```mode```. As ```type``` is key word,  it will break lint.